crates/ef-testing/src/storage/contract.rs

```diff
@@ -1,61 +1,47 @@
-use std::collections::HashMap;
-
 use hive_utils::madara::utils::genesis_set_bytecode;
 use reth_primitives::Bytes;
 use starknet::core::types::FieldElement;
 use starknet_api::{hash::StarkFelt, state::StorageKey as StarknetStorageKey};
 
-use crate::{models::error::RunnerError, utils::starknet::get_starknet_storage_key};
+use crate::models::error::RunnerError;
 
-use super::{write_is_initialized, write_madara_to_katana_storage};
+use super::{get_evm_address, get_is_initialized, get_starknet_storage, madara_to_katana_storage};
 
 /// Initializes the contract account.
 /// Writes the bytecode and the owner to a hashmap.
 pub fn initialize_contract_account(
     kakarot_address: FieldElement,
-    starknet_address: FieldElement,
     evm_address: FieldElement,
     bytecode: &Bytes,
-    destination: &mut HashMap<StarknetStorageKey, StarkFelt>,
-) -> Result<(), RunnerError> {
-    write_evm_address(evm_address, destination)?;
-    write_is_initialized(destination)?;
-    write_bytecode(starknet_address, bytecode, destination)?;
-    write_owner(kakarot_address, destination)
-}
-
-/// Writes the EVM address to a hashmap.
-fn write_evm_address(
-    evm_address: FieldElement,
-    destination: &mut HashMap<StarknetStorageKey, StarkFelt>,
-) -> Result<(), RunnerError> {
-    let evm_address_key = get_starknet_storage_key("evm_address", &[], 0)?;
-    destination.insert(evm_address_key, Into::<StarkFelt>::into(evm_address));
-    Ok(())
+) -> Result<Vec<(StarknetStorageKey, StarkFelt)>, RunnerError> {
+    let mut contract_storage = vec![
+        get_evm_address(evm_address)?,
+        get_is_initialized()?,
+        get_owner(kakarot_address)?,
```

Comment:
could we rename to

```
evm_address_storage(...
is_initialiazed_storage(...
owner_storage(...
```

(btw I thiink that the `get_` is "banned from rust, cc @Eikix)

Reply:
Yeah indeed decided to add the `get_` since I had the same understanding that `get_` is banned for getters. Can remove but I think `get` adds context in this case.